models/lfs_lock.go

```diff
@@ -12,27 +12,38 @@ import (
 	"time"
 
 	api "code.gitea.io/sdk/gitea"
+	"github.com/ngaut/log"
```

Comment:
import format: see https://github.com/go-gitea/gitea/blob/master/CONTRIBUTING.md#styleguide

Reply:
@ethantkoenig sorry I pass it as WIP seen I want to setup test for various git access to make that I don't break anything in future. It is my editor that setup this import, I will review my code once it works and remove WIP when ready. Sorry for the distrurb.

Reply:
Fixed.